Cargo.toml

```
@@ -0,0 +1,20 @@
+[package]
+name = "raw-validation-policy"
+version = "0.1.0"
+authors = ["Fabrizio Sestito <fabrizio.sestito@suse.com>"]
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies]
+k8s-openapi = { version = "0.17.0", default_features = false, features = [
+  "v1_25",
+] }
+kubewarden-policy-sdk = "0.8"
```

In this way we drop the k8s-openapi dependency of our SDK:

```suggestion
kubewarden-policy-sdk = { version = "0.8", default-feautures = false }
```

In case you wonder, this is a dependency dragged by the context aware API